.github/actions/build-frontend/action.yml

```diff
@@ -36,17 +35,8 @@ runs:
         yarn build:production
       shell: bash
 
-    - name: Build release for staging branch (conditional check)
-      if: inputs.env-name == 'staging'
-      working-directory: frontend-react
-      run: |
-        echo "" >> .env.staging
-        echo "REACT_APP_APPINSIGHTS_KEY=${{ inputs.appinsights-staging-key }}" >> .env.staging
-        yarn build:staging
-      shell: bash
-
     - name: Build release for non-production branch (conditional check)
-      if: (!(inputs.env-name == 'staging' || inputs.env-name == 'prod'))
+      if: inputs.env-name != 'prod'
```

Comment:
@doug-s-nava Some insight? This was a part of our test runner issue.

Reply:
good question, and the first part of the answer is I have no idea. Didn't even know there was a test branch!

second part of the answer is that this change was made largely so that tests would run when opening a PR against master. With the logic that was in place, there were two non-prod paths for the `build-frontend` action:
* run `yarn build:staging with .env.staging setup (when a PR faces `master`)
* run `yarn build:staging with .env.staging setup AND run linting and tests (when PR faces anything other than `master` or `production`

I was definitely making some assumptions on this change, such as that we actually do want to run testing and linting on `master` facing PRs, and it makes sense to me if we do want a separate case to handle a merge to `test`. Given what's here, maybe that would be something like:

* run `yarn build:test with .env.test setup AND run linting and tests

Side note: if it was set up this way, why was Github actions running `build-frontend` on PRs? It looks like there was / is a bug somewhere. You can see more here, including a screenshot: https://github.com/CDCgov/prime-reportstream/pull/5682